react/src/components/MapControlBar/MapControlbar.tsx

```diff
@@ -38,14 +39,19 @@ const CoordinatesDisplay = () => {
 
 interface Props {
   /**
-   * Active project
+   * Active project (info for Taggit link).
    */
   activeProject: Project;
 
   /**
    * Whether or not the map project is a public view.
    */
   isPublicView: boolean;
+
+  /**
+   * Placeholder property for use when declared later.
+   */
+  designSafeProject?: object;
```

Comment:
can this be dropped?

Reply:
If I drop that, the component test fails because the interface does not declare the property.

Reply:
designSafeProject isn’t used in the component, so it can be removed here and also where it’s passed as a prop in the unit test.

Reply:
Lines 80, 129, 131, 132. It is dynamically declared on line 80, but because it IS used in the component, the test strict checking complaings because it is not declared in the Interface. Adding that optional declaration made the test pass.

Reply:
It's being used in a unit test (line 104 of react/src/components/MapControlBar/MapControlbar.test.tsx). Can be dropped there and here 👍